lib/components/GalleryCard/GalleryCard.module.scss

```diff
@@ -25,27 +25,21 @@ $card-fixed-content-height: $gallery-card-height - $gallery-card-background-heig
     &:global(.selected) {
       .card {
         box-shadow: var(--depth-3);
-        border: 3px solid var(--color-border-focus);
+        border: 3px solid var(--color-accent);
       }
     }
 
     &:not(:global(.selected)):active {
       .card {
-        border: 1px solid var(--color-border-selected);
-        box-shadow: var(--depth-4);
-      }
-    }
-
-    &:not(:global(.selected)):hover {
-      .card {
-        border: 1px solid var(--color-border-tile-hover);
+        border: 1px solid var(--color-accent-selected);
         box-shadow: var(--depth-4);
       }
     }
 
+    &:not(:global(.selected)):hover,
     &:not(:global(.selected)):focus {
       .card {
-        border: 1px dashed var(--color-border-tile-hover);
+        border: 1px solid var(--color-accent);

```

Comment:
I think the pattern everywhere else is `dashed` on focus, and maybe `solid` for hover. Why is this changing?

Reply:
this is the border, not the outline... the gallery card has a solid border all the time, it just changes colors depending the state... we could make it so that on focus is dashed just to match the behavior of all other clickable items.